cpu/ppc64_cpu_test.py

```diff
@@ -53,9 +53,11 @@ def setUp(self):
         self.smt_values = {1: "off"}
         self.key = 0
         self.value = ""
-        self.max_smt_value = 4
-        if cpu.get_cpu_arch().lower() == 'power8':
-            self.max_smt_value = 8
+        self.max_smt_value = 8
+        if cpu.get_cpu_arch().lower() == 'power7':
+            self.max_smt_value = 4
+        if cpu.get_cpu_arch().lower() == 'power6':
+            self.max_smt_value = 2
```

Comment:
@sacsant @PraveenPenguin I am fine if we have a placeholder / skip for power9, and later add it when the hardware comes ?